src/app/modules/game/pages/lobby/lobby.component.scss

```diff
@@ -1,9 +1,117 @@
+@import "plugins/variables";
+@import "plugins/mixins";
+
 .lobby {
+  width: 100%;
+  &__container {
+    background-image: url(/assets/backgrounds/lobby-background.gif);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+    top: 0;
+    margin: 0;
+    padding: 0;
+    background-size: 90% 100%;
+    height: 100vh;
+    position: relative;
+  }
+  &__back {
+    padding: 0.2rem 1rem;
+    cursor: pointer;
+    width: 100px;
+    &:hover {
+      color: $primary-color--dark;
+    }
+  }
+  &__sidebar {
+    width: 20%;
+    top: 0;
+    right: 0;
+    min-height: 100vh;
+    z-index: 1;
+    background-color: $primary-color--dark;
+    border-left: 3px solid $black;
+    position: absolute;
+  }
+  &__chat {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    margin-left: 0.5rem;
+  }
   &__users {
     display: flex;
+    flex-direction: column;
     list-style-type: none;
+    justify-content: space-around;
+    text-align: center;
+    margin: 3rem auto;
+  }
+  &__button {
+    text-align: center;
+    background-color: $white;
+    font-weight: bold;
+    font-size: 4rem;
+    border-radius: 10px;
+    margin: 1rem;
+    padding: 0.5rem 1rem;
+    width: 250px;
+    border: 3px solid $black;
+    position: absolute;
+    top: 40%;
+    right: 30%;
+    transform: rotate(10deg);
+  }
+  .is-wait {
+    color: $quaternary-color--darker;
+    font-family: $ff-header-solid;
+    @include text-stroke();
+    cursor: auto;
+  }
+  .is-start {
+    color: #b8e088;
+    font-family: $ff-header-solid;
+    @include text-stroke();
+    &:hover {
+      color: darken(#b8e088, 10%);
+    }
+  }
+}
+
+.join {
+  width: 250px;
+  background: $quaternary-color;
+  border: 3px solid $black;
+  text-align: center;
+  padding: 0.5rem;
+  margin: 0.2rem 30%;
```

Comment:
looks strange to mix different units of measuring in margin

Reply:
% unit is more flexible to screen size